src/scss/components/_newsDescription.scss

```diff
@@ -0,0 +1,49 @@
+.description__container{
```

Comment:
You can nest this class inside the base `.description` class, like this:
```
.description {
  &__container {
    padding: 60px 20px;
  }
}
```

Reply:
Done!